components/src/main/res/values/attrs.xml

```diff
@@ -253,4 +253,15 @@
         <attr name="color2" format="color" />
     </declare-styleable>
 
+    <declare-styleable name="EditableListView">
+        <!-- Title to display. -->
+        <attr name="title" />
+        <!-- button Title to display. -->
+        <attr name="buttontitle" />
+        <!-- Button Text to display. -->
+        <attr name="editButtonText" />
+        <!-- Icon to display. -->
+        <attr name="icon" />
```

Comment:
I don't think you're using all of these? 
I think we do need to provide the "title", "button start editing text", "button finish editing text" and the "edit button text". We'll probably also need content description text for the edit button, as I think the first implementation of it has an "opens in your web browser" accessibility requirement.